src/H5ES.c

```diff
@@ -269,7 +269,7 @@ H5ESget_requests(hid_t es_id, H5_iter_order_t order, hid_t *connector_ids, void
     herr_t  ret_value = SUCCEED; /* Return value */
 
     FUNC_ENTER_API(FAIL)
-    H5TRACE5("e", "iIo*i**xx", es_id, order, connector_ids, requests, count);
+    H5TRACE6("e", "iIo*i**xzx", es_id, order, connector_ids, requests, array_len, count);
```

Comment:
This change is necessary because of a change from HD??

Reply:
It's from a prior commit where someone didn't run bin/trace via autogen.sh